Nbuild/Program.cs

```diff
@@ -71,6 +71,7 @@ static int Main(string[] args)
                         Cli.CommandType.git_autotag => Command.SetAutoTag(options.BuildType),
                         Cli.CommandType.git_push_autotag => Command.SetAutoTag(options.BuildType, push: true),
                         Cli.CommandType.git_branch => Command.DisplayGitBranch(),
+                        Cli.CommandType.git_clone => Command.Clone(options),
```

Comment:
Change Command.Clone's parameters to Cli options so that we can pass the path of the clone operation.